cmd/kubelet/app/options/options.go

```diff
@@ -179,9 +179,10 @@ type KubeletFlags struct {
 	// schedulable. Won't have any effect if register-node is false.
 	// DEPRECATED: use registerWithTaints instead
 	RegisterSchedulable bool
-	// RequireKubeConfig is deprecated! A valid KubeConfig is now required if --kubeconfig is provided.
-	RequireKubeConfig bool
 	// nonMasqueradeCIDR configures masquerading: traffic to IPs outside this range will use IP masquerade.
+	// RequireKubeConfig is deprecated! Keep it because pull-kubernetes-e2e-kops-aws needs it.
+	// TODO: when pull-kubernetes-e2e-kops-aws does not need this parameter, delete it.
+	RequireKubeConfig bool
```

Comment:
You can drop the bool from the config struct and bind it to a local bool var and leave it noop